common/supplemental/supplementalData.xml

```diff
@@ -5436,15 +5436,11 @@ XXX Code for transations where no currency is involved
 		<parentLocale parent="zh_Hant_HK" locales="zh_Hant_MO"/>
 	</parentLocales>
 
-	<parentLocales component="segmentations">
-		<parentLocale parent="zh" locales="zh_Hant"/>
+	<parentLocales component="segmentations grammaticalFeatures plurals">
 	</parentLocales>
 
 	<parentLocales component="collations">
-		<parentLocale parent="bs" locales="bs_Cyrl"/>
-		<parentLocale parent="sr" locales="sr_Latn"/>
 		<parentLocale parent="sr_ME" locales="sr_Cyrl_ME"/>
-		<parentLocale parent="zh" locales="zh_Hant"/>
```

Comment:
Do we need/want to add a `<parentLocale parent="root" locales="nonlikelyScript"/>` since the rules no longer inherit? If not, we are changing behavior; for example, `az-Arab` will now inherit from `az` instead of from root for the purpose of collation data.

Reply:
Yes, we are changing behavior, but it is to what it should have been. Collation, segmentation, grammar, etc don't want to skip over the base language — that was a mistake in the original addition of splitting parentLocales by component. We found that out when we delved deeper into what ICU was doing, and realized that what it was doing was the behavior that we really wanted.

(they may also have customizations of their own, of course).